cmd/server/cadence/fx_test.go

```diff
@@ -50,24 +49,3 @@ func TestFxDependencies(t *testing.T) {
 		Module)
 	require.NoError(t, err)
 }
-
-func TestFxStart(t *testing.T) {
```

Comment:
This won't work since cli hardcode gocql.Quorum consistency, which won't work for a single cassandra setup.
Most of this is tested in server_test.go

I've tried running locally cadence-server start without any issues.
However, we probably need to fix this.